scripts/issue/245/issue245.py

```diff
@@ -0,0 +1,26 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from brownie import interface
+from rich.console import Console
+
+
+console = Console()
+
+NEW_IMPLEMENTATION = "0x67Db14E73C2Dce786B5bbBfa4D010dEab4BBFCF9"
```

Comment:
confirmed that all other f22tokens have this address as their implementation.

also seen that it has been deployed months ago, i wonder how we ended up with the old version of the implementation for fbvecvx...?